cfgov/legacy/static/knowledgebase/less/notification.less

```diff
@@ -0,0 +1,102 @@
+/* ==========================================================================
+   Capital Framework
+   Notifications Styling
+   ========================================================================== */
+
+// Theme variables
+
+@base-font-size-px:           @16px;
```

Comment:
Less compilation fails on this. Should just be `16px`, no `@`.

Reply:
I fixed that locally and got the Less to build, but I'm still seeing the large icon locally. You're not seeing it? You have to scroll past the main content of the page.

Reply:
No, make sure you run `gulp clean && gulp build`.

Reply:
Got it. May have also been browser cache not clearing.

Reply:
This value still needs correcting, though.